src/components/ImagesCarousel.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+
+const ImagesCarousel = () => (
+  <View>
+    <Text style={styles.headingText}>{'DESCRIPTION'}</Text>
+    <Image source={require('../../assets/Sample_Business_Image.png')} style={{ width: 250, height: 300, borderRadius: 10, alignSelf: 'center' }} resizeMode="contain" />
```

Comment:
This is just a placeholder/prototype for now. Since this feature has not been implemented server-side and was not due to be implemented in sprint 1 I am leaving this for now.